modules/assim.sequential/R/sda.enkf.R

```diff
@@ -739,8 +735,48 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL) {
       enkf.params[[t]] <- list(mu.f = mu.f, Pf = Pf, mu.a = mu.a, Pa = Pa)
     }
     
-    ## update state matrix
-    analysis <- as.data.frame(rmvnorm(as.numeric(nens), mu.a, Pa, method = "svd"))
+    ###-------------------------------------------------------------------###
```

Comment:
Glad to have ensemble adjustment added, and I agree it should be the default, but I wonder if we want to retain a flag and an ```if``` to toggle between the two options rather than simply deleting the original EnKF?

Reply:
👍

Reply:
I added the flag but for some reason it's not changing here

Reply:
Did you forget to make a commit or to push it? I don't see the flag

Reply:
yeah it's in one of the commits below https://github.com/PecanProject/pecan/pull/1605/commits/58112a0e4285cd77ee4f98438cbb5ce927f7c045
But maybe there is more that I need to change?

Reply:
Hmm -- very odd. Could you just try re-adding the changes? I wonder if in some combo of merging you accidentally overwrote the change?